consensus/ethash/algorithm_go1.8.go

```diff
@@ -27,10 +27,14 @@ import "math/big"
 func cacheSize(block uint64) uint64 {
 	// If we have a pre-generated value, use that
 	epoch := int(block / epochLength)
-	if epoch < len(cacheSizes) {
+	if epoch < maxEpoch {
 		return cacheSizes[epoch]
 	}
 	// No known cache size, calculate manually (sanity branch only)
+	return calcCacheSize(epoch)
+}
+
+func calcCacheSize(epoch int) uint64 {
```

Comment:
Please document this method. I know it's private, but all methods in ethash are fully documented and I'd like to keep it so.